attributes/default.rb

```diff
@@ -2,6 +2,9 @@
 default['packagecloud']['gpg_key_path'] = "/gpg.key"
 default['packagecloud']['hostname'] = nil
 
+default['packagecloud']['_stored_hostname'] = nil
+default['packagecloud']['override_hostname'] = nil
```

Comment:
What would be the difference between `node['packagecloud']['hostname']` and `node['packagecloud']['override_hostname']` be now?